source/uwp/winui3/NuGet/AdaptiveCards.ObjectModel.WinUI3.nuspec

```diff
@@ -0,0 +1,46 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>AdaptiveCards.ObjectModel.WinUI3</id>
+        <version>1.0.0</version>
+        <authors>Microsoft</authors>
+        <summary>Adaptive Cards Object Model library for WinUI3</summary>
+        <description>Used to work with the object model for Adaptive Cards</description>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <tags>adaptivecards;adaptive-cards</tags>
+        <projectUrl>https://adaptivecards.io</projectUrl>
+        <iconUrl>https://adaptivecards.io/content/icons_blue/blue-48.png</iconUrl>
+        <license type="file">EULA-Windows.txt</license>
+        <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    </metadata>
+    <files>
+        <file src="..\$Configuration$\AdaptiveCardsObjectModel\AdaptiveCards.ObjectModel.WinUI3.winmd" target="lib\uap10.0"/>
+        <file src="..\$Configuration$\AdaptiveCardsObjectModel\AdaptiveCards.ObjectModel.WinUI3.winmd" target="lib\native"/>
```

Comment:
no need for this :)
you can reference uap10/.winmd from native targets:)

Reply:
If that's the case, should we remove this line from the UWP version?